src/test/java/com/exactpro/junit5/IllegalArgumentFormatException.java

```diff
@@ -1,7 +1,9 @@
 package com.exactpro.junit5;
 
-class IllegalArgumentFormatException extends IllegalArgumentException {
-    IllegalArgumentFormatException(String errorMessage) {
+import java.io.IOException;
+
+class IllegalArgumentFormatException extends IOException {
```

Comment:
It shouldn't be IOException. We throw it when hex string passed to our annotation is invalid. No IO happens at this moment. It's purely mistake made by developer. I think we can simply use standard 'InvalidArgumentException` for that